test/suites/integration/subnet_test.go

```diff
@@ -108,3 +138,43 @@ func getSubnets(tags map[string]string) map[string][]string {
 	Expect(err).To(BeNil())
 	return subnets
 }
+
+func enableResourceNameDNSArecordForAllSubnets(subnetIDs ...string) {
+	for subnetID := range subnetIDs {
+		inputOne := &ec2.ModifySubnetAttributeInput{
+			EnableResourceNameDnsARecordOnLaunch: &ec2.AttributeBooleanValue{
+				Value: aws.Bool(true),
+			},
+			SubnetId: aws.String(subnetIDs[subnetID]),
+		}
+
+		inputTwo := &ec2.ModifySubnetAttributeInput{
+			PrivateDnsHostnameTypeOnLaunch: aws.String("resource-name"),
+			SubnetId:                       aws.String(subnetIDs[subnetID]),
+		}
+
+		_, err := env.EC2API.ModifySubnetAttribute(inputOne)
+		Expect(err).To(BeNil())
+		_, err = env.EC2API.ModifySubnetAttribute(inputTwo)
+		Expect(err).To(BeNil())
+	}
+}
+
+func disabledResourceNameDNSArecordForAllSubnets(subnetIDs ...string) {
+	for subnetID := range subnetIDs {
+		inputOne := &ec2.ModifySubnetAttributeInput{
+			EnableResourceNameDnsARecordOnLaunch: &ec2.AttributeBooleanValue{
+				Value: aws.Bool(false),
+			},
+			SubnetId: aws.String(subnetIDs[subnetID]),
+		}
+		inputTwo := &ec2.ModifySubnetAttributeInput{
+			PrivateDnsHostnameTypeOnLaunch: aws.String("ip-name"),
+			SubnetId:                       aws.String(subnetIDs[subnetID]),
+		}
```

Comment:
rather than setting these as variables, can you inline these in the ModifySubnetAttribute arguments, like this?

```
		_, err := env.EC2API.ModifySubnetAttribute(&ec2.ModifySubnetAttributeInput{
			EnableResourceNameDnsARecordOnLaunch: &ec2.AttributeBooleanValue{
				Value: aws.Bool(false),
			},
			SubnetId: aws.String(subnetIDs[subnetID]),
		})
		Expect(err).To(BeNil())
``` 

and same for the enable func above.